### PR TITLE
adding files for user-information feature

### LINKED ISSUE #1: Ability to add users information to the top of the resume
https://github.com/MLH-Fellowship/orientation-project-python/issues/22

### PR DESCRIPTION
**Pull request by anreetka**
## Description
- Added UI component for the user to be able to add, view and edit user information on top of the resume

## Testing
1. When the user sees the screen, it should be show screen when user has no data and and a button with Add user details. 
2. On clicking add user details, a new form appears where the user can enter details and submit. If the user enters incorrect format, the user should see respective errors before successfully submitting data to the backend.
3. After the user has filled out the information, the user should be able see the information under User Information and be able to edit the information which should load the form again and update data on clicking submit.

## Screenshots
- When there's no user data:
image

- When the user is adding data
image

- Errors if the data is invalid:
image

- When the user has added data :
image

**Note**: edit button should show the form with populated data that the user already filled out.

## What Changed
- created UI component folder for user-information under ```src/features```
- added validation regex for email_address and phone_number in index.js under constants 
- Imported and utilized user ```<UserInformationComponent/>``` in ```App.js``` within html content

Fixes #1

### CODE REVIEW

File: src/features/user-information/AddUserInformationComponent.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    if (!formData.name) {
+      newErrors.name = "Name is required!";
+    }
+
+    const internationalPhoneFormat = /^\+\d{1,3}\d{7,14}$/;
+    if (internationalPhoneFormat.test(formData.phone_number)) {
+      newErrors.phone_number = "Please enter a valid phone number";
+    }
+
+    const emailFormat = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
```
Comment (Pradyuman7) [inline src/features/user-information/AddUserInformationComponent.js:44]: This and above regex might be reusable, let's move it to constants

**Review by Pradyuman7 — COMMENTED** [review]
Looks good, one small comment, and it might be a good idea to add a few ss or videos of the new changes so it's easier to refer

File: src/features/user-information/AddUserInformationComponent.js (hunk 1 above)
Comment (anreetka) [reply src/features/user-information/AddUserInformationComponent.js]: I have updated the regex to handle more test cases for phone number format of user input and moved it to constants/index.js. It's available for the code review, thanks! :)

**Review by Pradyuman7 — APPROVED** [review]
LGTM 💯

**Pull request merged**